frameworklauncher/src/main/java/com/microsoft/frameworklauncher/common/model/TaskStatus.java

```diff
@@ -41,6 +41,7 @@
   // containerIp is the assigned ipv4 address of the corresponding containerHost
   private String containerIp;
   // containerGpus is the assigned GpuAttribute of the container
+  private String containerPorts;
```

Comment:
>containerPorts [](start = 17, length = 14)

Should be containerPortRanges

Reply:
GPU, ports, IPs, Host, they should naming consistent. 

---
In reply to: [167156972](https://github.com/Microsoft/pai/pull/318#discussion_r167156972) [](ancestors = 167156972)